src/main/java/com/aws/greengrass/util/orchestration/SystemServiceUtilsFactory.java

```diff
@@ -43,6 +44,15 @@ public synchronized SystemServiceUtils getInstance() {
         } catch (IOException e) {
             logger.atError().log("Unable to determine init process type");
```

Comment:
My comment was maybe a bit wrong, this shouldn't be an error yet because we haven't identified that the system _isn't_ procd, so we still may be able to determine the init type.